bindings/ruby/lib.rs

```diff
@@ -0,0 +1,198 @@
+extern crate rutie;
+extern crate limbo_core;
```

Comment:
The `extern crate` thing is an old Rust thing. Once you switch newer edition in `Cargo.toml` these go away